python/GaelZllHbb13TeVconfig/datacards.ini

```diff
@@ -283,7 +283,7 @@ dcName = ZmmCRZlight_13TeV
 cut = CRZlight
 signal = ZH_HToBB_ZToLL_M125_pow ggZH_HToBB_ZToLL_M125_pow
 dcBin = CRZlight
-data = data_SM_Run2015C_25ns_16Dec2015 data_SM_Run2015D-16Dec2015
+data = data_SM_Run2015D_25ns_16Dec2015 data_SM_Run2015D-16Dec2015
```

Comment:
I am afraid it was correct before and now is wrong,
as I see that 2 datasets are specified, and now they are the same apart from small differences in the name,
while one should correctly include the Run2015C and Run2015D
my bad!